doc/api/v0.yaml

```diff
@@ -136,6 +138,27 @@ paths:
         "404":
           description: The requested challenge was not found
 
+  /api/v0/reviews/{proposal_id}:
```

Comment:
It's a bit weird that we're adding these `/v0` paths when we have a v1 API alongside.

Or was the versioning meant to apply piecemeal to the paths, `/v0/reviews` meaning "version 0 of the reviews endpoint"?

Reply:
It is meant to be the Api version.

Reply:
So it should be `/v1/reviews` then?

Reply:
Well, we do not have a `v1` per se. We use v0 permanently (at least for now) since API it is changed anyway from fund to fund. And, we do not need to support api changes for now. Or should we?